workspaces/arborist/lib/arborist/index.js

```diff
@@ -82,7 +140,10 @@ class Arborist extends Base {
       installStrategy: options.global ? 'shallow' : (options.installStrategy ? options.installStrategy : 'hoisted'),
       lockfileVersion: lockfileVersion(options.lockfileVersion),
       packageLockOnly: !!options.packageLockOnly,
-      packumentCache: options.packumentCache || new Map(),
+      // XXX: Does it make a difference to create the packumentCache as a global singleton?
+      // Meaning: do we want to share this cache between all arborist constructors or not?
+      // If so we could move this to the CLI and pass it down when we create new arborist instances
```

Comment:
The cache is currently unaware of packument type, it would currently risk mixing packuments with corgis.

Reply:
We have so many caching layers in npm, surely some of them are not helping as much as we think.